lmdeploy/vl/templates.py

```diff
@@ -86,6 +86,9 @@ def convert_messages(self, messages, sequence_start=True):
             role = message['role']
             content = message['content']
             if role != 'user' or isinstance(content, str):
+                if role == 'assistant' and isinstance(content, list):
```

Comment:
can we make ut to verify the vlm chat templates?

Reply:
If we use ut to verify the function, we may install the 3rd library like llava or copy their code.

Reply:
we can make some fixed input and expected output
Let our vl template process the input, verifying whether the output equals to the expected output